### ISSUE TITLE
Quarto Preview Uses Default Shell Instead of Functional Shell

### ISSUE THREAD

**Issue by emstruong** [posit-dev/positron#5859]
#### Positron and OS details:

Positron Version: 2025.01.0 (system setup) build 39
Code - OSS Version: 1.93.0
Commit: c7ef34fc7078ac7ecf13ff3328605dfba93f3eea
Date: 2024-12-09T02:44:40.379Z
Electron: 30.4.0
Chromium: 124.0.6367.243
Node.js: 20.15.1
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.26100
#### Interpreter details:

R 4.4.0

## Describe the issue:

If your default terminal in Positron is WSL and you try to render quarto preview, Positron will try to use WSL to render the document as opposed to Powershell. (By default and with a regular installation on Windows, Quarto would only be accessible via Powershell and not WSL)

## Expected or desired behavior:

Either the shell for each program could each be manually set or everything is set to one fixed terminal.

**Comment by jmcphers** [comment]
Do you also see this behavior when using the Quarto extension with Visual Studio Code?

**Comment by juliasilge** [comment]
Please let us know if you have further questions!

**Comment by emstruong** [comment]
@jmcphers Yes, it seems to happen even in VS Code. The option to have WSL as default seems to be under 'Terminal -> Integrated -> Default Profile: Windows'.

I guess this is an issue for the Quarto repo?
